appcenter-data/ios/AppCenterReactNativeData/AppCenterReactNativeData.m

```diff
@@ -28,16 +28,22 @@
 #import <AppCenterData/MSDocumentWrapper.h>
 #import <AppCenterData/MSReadOptions.h>
 #import <AppCenterData/MSWriteOptions.h>
+#import <AppCenterData/MSPaginatedDocuments.h>
+#import <AppCenterData/MSPage.h>
 #import <AppCenterReactNativeShared/AppCenterReactNativeShared.h>
 
 @interface AppCenterReactNativeData () <RCTBridgeModule>
 
+@property (nonatomic) NSMutableDictionary *paginatedDocuments;
```

Comment:
Let's add `strong` property attribute to emphasize it should be `strong` instead of `copy`, though `strong` is the default attribute. We do the same in App Center Apple SDK to emphasize we should use one property attribute over the other.

```suggestion
@property (nonatomic, strong) NSMutableDictionary *paginatedDocuments;
```

Reply:
Added